README.md

```diff
@@ -72,17 +72,42 @@ ENV SPRING_PROFILES_ACTIVE docker,mysql
 In the `mysql section` of the `application.yml` from the [Configuration repository], you have to change 
 the host and port of your MySQL JDBC connection string. 
 
+## Custom metrics monitoring
+
+@todo Add default custom dashboards to grafana
+
+Grafana and Prometheus are included in the `docker-compose.yml` configuration, and the public facing applications have been instrumented with [MicroMeter](https://micrometer.io) to collect JVM and custom business metrics.
```

Comment:
For development, does the application work if Grafana and Prometheus are not started?

Reply:
There is no dependency on those services at runtime. This implementation produces an additional endpoint on the actuator which is scraped by prometheus.